web_easy_switch_company/README.rst

```diff
@@ -0,0 +1,88 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+web_easy_switch_company
+=======================
+
+This module extend web backend and allow user to switch to another company more easyly.
```

Comment:
This module extends web backend and allows user to switch to another company more easily.